### PR TITLE
Allow user to input custom tractography

### PR DESCRIPTION
**Pull request by 36000**
- [x] Consider moving test to nightly

### CODE REVIEW

**Comment by 36000** [comment]
@arokem I think this is ready to go

File: AFQ/tests/test_api.py [hunk 1]
```diff
@@ -138,6 +138,46 @@ def create_dummy_bids_path(n_subjects, n_sessions):
...
+        op.join(
+            op.expanduser('~'),
+            'AFQ_data',
+            'stanford_hardi_tractography',
+            'tractography_subsampled.trk'),
+        op.join(
+            tmpdir.name,
+            'stanford_hardi',
+            'derivatives',
+            'vistasoft',
```
Comment (arokem) [inline AFQ/tests/test_api.py:165]: Does the tractography have to reside in the 'dmriprep' derivatives folder?  Can it be in another folder (i.e., generated by some other pipeline)?

Comment (36000) [reply AFQ/tests/test_api.py:165]: Good point. Scope should be specified by the user filters instead.

Comment (36000) [reply AFQ/tests/test_api.py:165]: I made this change in "make scope a user choice" commit. Does this look like a good solution?

**Comment by 36000** [comment]
@arokem I think I addressed the earlier review. This is ready for another review or merge.

**Comment by arokem** [comment]
Yep. I think that we might want to provide better documentation and maybe even an example in the gallery for how this is done. I'll add an issue to address this on a separate PR.

**Pull request merged**